src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -5640,18 +5946,31 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
 
       if (rk_step == 1) then
 
-         delsq_theta(1:nVertLevels,cellStart:cellEnd) = 0.0
-
+         !$acc parallel default(present)
+         !$acc loop gang worker
          do iCell=cellStart,cellEnd
-            tend_theta_euler(1:nVertLevels,iCell) = 0.0
+
+            !$acc loop vector
+            do k=1,nVertLevels
+               delsq_theta(k,iCell) = 0.0_RKIND
+            end do
+
+            !$acc loop vector
+            do k=1,nVertLevels
+               tend_theta_euler(k,iCell) = 0.0_RKIND
```

Comment:
Was there any reason to put `tend_theta_euler` and `delsq_theta` into separate loops? I realize they were slightly different operations before these changes, but I'm not seeing a reason now to keep them separate.

Reply:
That's fair. I'll combine the initialization of these two variables.